### ISSUE TITLE
One-off Cost to unlock portals?

### ISSUE THREAD

**Issue by thunerb** [Multiverse/Multiverse-Portals#260]
I was thinking a nice idea would be to make portals cost only once and then players are able to use it as much as they'd like. Idea as we have one nether portal in spawn due to griefing issues on the server via portals and so it'd be nice not to give it out for free but without incurring a cost every time the portal is used.

**Comment by FernFerret** [comment]
Sounds like @moddinglegend01 is onto something. I'd say that this request is out of scope for us, at least at this time. Thanks for the suggestion though!
